### ISSUE TITLE
Dense tab titles temporarily expands while italic

### ISSUE THREAD

**Issue by jukzi** [eclipse-platform/eclipse.platform.ui#1705]
Example: i tried to search call hierarchy of JavaPluginImages.DESC_OBJS_REFACTORING_WARNING
That temporarily moves the whole search result one line to the bottom, because the tab title is shown in italics for a short moment, expanding tabs size, pushing other elements away. That leaves a disturbing impression:

![italic](https://github.com/eclipse-platform/eclipse.platform.ui/assets/51790620/34f1eaf7-1433-46c8-9f13-e8c0ff39b6ca)

**Comment by jukzi** [comment]
Version: 2024-03 (4.31)
Build id: I20240218-1800
@HeikoKlare i guess this is a regression of recent tab title changes

**Comment by iloveeclipse** [comment]
Italicfont is shown for "busy" view tabs, that is not new. Tab size changes if different font is set, which is also not new. So I think thisis not a regression.

**Comment by jukzi** [comment]
you are right,  i can reproduce it with older version: One has to have very unlucky pixel exact width of all tab titles to have the move column down effect.

**Comment by HeikoKlare** [comment]
As @iloveeclipse already pointed out, this is not a regression but existing behavior. For example in 2023-03, it looks as follows:
![Animation](https://github.com/eclipse-platform/eclipse.platform.ui/assets/755472/30063a66-b9ab-4d5d-b478-ebde1375b441)
